lib/mayaUsd/render/vp2RenderDelegate/material.cpp

```diff
@@ -1877,6 +1895,8 @@ MHWRender::MShaderInstance* HdVP2Material::_CreateMaterialXShaderInstance(
         // Add automatic tangent generation:
         shaderInstance->addInputFragment("materialXTw", "Tw", "Tw");
 
+        shaderInstance->setIsTransparent(ogsFragment.isTransparent());
```

Comment:
This requires getting at least https://github.com/autodesk-forks/MaterialX/pull/1285 so the transparency is correctly computed.

Reply:
It sounds like we should update the build instructions in the doc with the above info as well?

Reply:
Done.